### PR TITLE
Feat more dysfunction improvement

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

**Review by Yoga07 — APPROVED** [review]

File: sn/src/node/api/dispatcher.rs [hunk 1]
```diff
@@ -186,9 +186,29 @@ impl Dispatcher {
                         .enqueue_and_handle_next_cmd_and_offshoots(cmd, None)
                         .await
                     {
-                        error!("Error sending a cleaning up unused PeerLinks: {:?}", e);
+                        error!(
+                            "Error sending Propose Offline for dysfunctional nodes: {:?}",
+                            e
```
Comment (davidrusu) [inline sn/src/node/api/dispatcher.rs:191]: You can put the `e` into the format string to cleanup this formatting if you'd like:
```suggestion
                            "Error sending Propose Offline for dysfunctional nodes: {e:?}"
```

**Review by davidrusu — APPROVED** [review]
Looks good

**Pull request merged**